pkg/kubectl/resource/visitor.go

```diff
@@ -375,7 +363,7 @@ func ignoreFile(path string, extensions []string) bool {
 }
 
 // FileVisitorForSTDIN return a special FileVisitor just for STDIN
-func FileVisitorForSTDIN(mapper *Mapper, schema validation.Schema) Visitor {
+func FileVisitorForSTDIN(mapper *Mapper, schema validation.Schema) visitorStreamTransformer {
```

Comment:
Seems like FileVisitorForSTDIN doesn't actually need to be public.  Any concerns about making it private?